src/components/MovieDetails/index.tsx

```
@@ -1143,7 +1137,574 @@ const MovieDetails = ({ movie }: MovieDetailsProps) => {
         hideWhenEmpty
       />
       <div className="extra-bottom-space relative" />
-    </div>
+      </div>
+      ) : (
+      /* ── AMOLED full-bleed layout ── */
```

Biome (2.4.6)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)